resources/views/pages/home.blade.php

```diff
@@ -0,0 +1,317 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Simael</title>
+
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&family=Stack+Sans+Headline:wght@200..700&display=swap" rel="stylesheet">
+    
+    @vite('resources/css/app.css')
+    @vite('resources/js/app.js')
+
+    <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
+
+    <style>
+        html {
+            scroll-behavior: smooth;
+        }
+        nav {
+            font-family: 'Stack Sans Headline', sans-serif;
+        }
+    </style>
+    
+</head>
+<body class="bg-gradient-to-br from-blue-800 to-cyan-700 text-white font-sans relative">
+    
```

Comment:
please extend from layouts.app, and DO NOT create html tags separately, this is useful to maintain consistency of navbar and title
```suggestion
@extends('layouts.app')

@section('content')
```